### PR TITLE
New pictures

### PR DESCRIPTION
**Pull request by Daniel-ND**

### CODE REVIEW

File: app/src/main/java/com/google/droidjump/GameConstants.java [hunk 1]
```diff
@@ -24,10 +24,10 @@
...
  */
 public interface GameConstants {
-    int DROID_COUNT_ON_FULL_DROID_PICTURE = 9; // Count of droids at res/drawable/droid.png image.
-    int DROID_JUMPING_CHARACTER_INDEX = 4; // Index of droid at res/drawable/droid.png image.
-    int DROID_FIRST_STEP_INDEX = 5; // Index of droid at res/drawable/droid.png image.
-    int DROID_SECOND_STEP_INDEX = 6; // Index of droid at res/drawable/droid.png image.
+    int DROID_COUNT_ON_FULL_DROID_PICTURE = 3; // Count of droids at res/drawable/droid.png image.
+    int DROID_JUMPING_CHARACTER_INDEX = 0; // Index of droid at res/drawable/droid.png image.
+    int DROID_FIRST_STEP_INDEX = 1; // Index of droid at res/drawable/droid.png image.
+    int DROID_SECOND_STEP_INDEX = 2; // Index of droid at res/drawable/droid.png image.
```
Comment (redls) [inline app/src/main/java/com/google/droidjump/GameConstants.java:30]: why are these changed?

**Review by redls — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/google/droidjump/GameConstants.java (hunk 1 above)
Comment (Daniel-ND) [reply app/src/main/java/com/google/droidjump/GameConstants.java:30]: Because in my picture there are three different positions of the droid. But the original one had more (we just didn't use them). Because of this, the indexes have changed.

**Review by redls — APPROVED** [review]
